### PR TITLE
Add bullet point icon

### PR DESCRIPTION
**Pull request by StefanoA1**
**Detailed purpose of the PR**

**Result and observation**

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

**Review by AdrieanKhisbe — APPROVED** [review]
LGTM

**Review by silouone — APPROVED** [review]

File: packages/@coorpacademy-nova-icons/third-party/nova-composition.iconjar/icons/bullet-point.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M1.556 8.361a1.556 1.556 0 1 0 0-3.111 1.556 1.556 0 0 0 0 3.111zM7 8.361A1.556 1.556 0 1 0 7 5.25a1.556 1.556 0 0 0 0 3.111zM12.444 8.361a1.556 1.556 0 1 0 0-3.111 1.556 1.556 0 0 0 0 3.111z" fill="#1D1D2B"/>
```
Comment (esa-coorp) [inline packages/@coorpacademy-nova-icons/third-party/nova-composition.iconjar/icons/bullet-point.svg:2]: tu ne pourras pas changer la couleur dans les composants 
```suggestion
    <path d="M1.556 8.361a1.556 1.556 0 1 0 0-3.111 1.556 1.556 0 0 0 0 3.111zM7 8.361A1.556 1.556 0 1 0 7 5.25a1.556 1.556 0 0 0 0 3.111zM12.444 8.361a1.556 1.556 0 1 0 0-3.111 1.556 1.556 0 0 0 0 3.111z" fill="currentColor"/>
```

Comment (esa-coorp) [reply packages/@coorpacademy-nova-icons/third-party/nova-composition.iconjar/icons/bullet-point.svg:2]: relance un `npm run prepare`

File: packages/@coorpacademy-nova-icons/src/components/nova-composition/coorpacademy/bullet-point.native.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import Svg, { Path } from "react-native-svg";
+
+const SvgComponent = (props) => (
+  <Svg viewBox="0 0 14 14" fill="none" {...props}>
+    <Path
+      d="M1.556 8.361a1.556 1.556 0 100-3.111 1.556 1.556 0 000 3.111zm5.444 0A1.556 1.556 0 107 5.25a1.556 1.556 0 000 3.111zm5.444 0a1.556 1.556 0 100-3.111 1.556 1.556 0 000 3.111z"
+      fill="#1D1D2B"
```
Comment (esa-coorp) [inline packages/@coorpacademy-nova-icons/src/components/nova-composition/coorpacademy/bullet-point.native.js:8]: couleur non dynamique

File: packages/@coorpacademy-nova-icons/src/components/nova-composition/coorpacademy/bullet-point.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+
+const SvgComponent = (props) => (
+  <svg viewBox="0 0 14 14" fill="none" {...props}>
+    <path
+      d="M1.556 8.361a1.556 1.556 0 100-3.111 1.556 1.556 0 000 3.111zm5.444 0A1.556 1.556 0 107 5.25a1.556 1.556 0 000 3.111zm5.444 0a1.556 1.556 0 100-3.111 1.556 1.556 0 000 3.111z"
+      fill="#1D1D2B"
```
Comment (esa-coorp) [inline packages/@coorpacademy-nova-icons/src/components/nova-composition/coorpacademy/bullet-point.js:7]: couleur non dynamique

**Review by esa-coorp — CHANGES_REQUESTED** [review]

**Review by esa-coorp — APPROVED** [review]

**Pull request merged**